KTXContainerFormat.md

```diff
@@ -0,0 +1,13 @@
+# KTX Container Format
+
+Universal Textures are comprised of two formats, written to a single file:
+
+1. **Compression format:** A compact representation of pixel data, in a layout designed for very efficient transcoding to one or more GPU compression formats. See ETC1S and UASTC, below.
+2. **Container format:** An explanatory wrapper around the compression format, describing how that data should be used by the client software.
+
+Without a container format, compressed data cannot be correctly moved between software applications. By default, the `basisu` utility produces `.basis` files, a custom container format understood by the Basis Universal reference transcoder. For more general interoperability, and particularly for use within portable 3D formats like [glTF 2.0](https://github.com/KhronosGroup/glTF), we instead recommend using the [KTX 2.0 Container Format](http://github.khronos.org/KTX-Specification/) (`.ktx2`). KTX supports the same underlying compression formats, but provides a more fully-specified container description, license protections under the Khronos IP Framework, and the likelihood of wider support from 3D authoring software over time.
```

Comment:
I've moved basisu / `.basis` to a footnote. Since this PR includes only a single section, I will leave the final decision of where that footnote should appear to be resolved when the section is merged into `DeveloperGuidelines.md` from #1.